res/controllers/Numark-Party-Mix.scripts.js

```diff
@@ -1,10 +1,15 @@
 var NumarkPartyMix = {};
 
 // jogwheel
-NumarkPartyMix.jogScratchSensitivity = 1024;
-NumarkPartyMix.jogScratchAlpha = 1; // do NOT set to 2 or higher
-NumarkPartyMix.jogScratchBeta = 1 / 32;
+// I am not sure if the jogwheel is behaving consistently,
+// or if my variables are wrong.
+// If one is scratching the point in a track is moving.
+// Maybe these variables need improvement.
```

Comment:
can you elaborate?

Reply:
if you put your finger on the jogwheel in scratch mode (without pressing play) and circle clockwise and one circle counter clockwise that the the track is at exactly the same position. this is unfortunately not the case. But I don't know if this is a problem of the hardware or my variables.

Reply:
Mhmm. You could do a test with a simple handler that just counts all the ticks from each jogwheel.

Reply:
someone who is interested in it can do that

Reply:
its ~298 the jump happens when you scratch back and forth fast around the same spot ... i guess its when you move too fast for the clock of the device, so it misses some ticks.

Reply:
Mhmm. So you can reproduce it by just summing the values the hardware sends?